src/lib/apexPmd.ts

```diff
@@ -140,18 +141,21 @@ export class ApexPmd {
       ...additionalClassPaths,
     ].join(CLASSPATH_DELM);
 
-    let env : NodeJS.ProcessEnv = {...process.env};
-    env["CLASSPATH"] = classPath;
+    let env : NodeJS.ProcessEnv = {};
+    env["CLASSPATH"] = `"${classPath}"`;
```

Comment:
That's the actual fix.
I've changed the log output as well to see more info in case there are further problems.